### PR TITLE
Fix SQL injections in ajax_table.php

### PR DESCRIPTION
**Pull request by Jellyfrog**
Thanks to Shielder (https://shielder.it/) for pointing this out.

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: html/ajax_table.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 settype($current, 'integer');
 $rowCount = $_REQUEST['rowCount'];
 settype($rowCount, 'integer');
-if (isset($_REQUEST['sort']) && is_array($_POST['sort'])) {
+if (isset($_REQUEST['sort']) && is_array($_REQUEST['sort'])) {
```
Comment (murrant) [inline html/ajax_table.php:28]: Guessing this was for testing?

Comment (Jellyfrog) [reply html/ajax_table.php:28]: Yes and no. It uses REQUEST elsewhere so why not.

Comment (SourceDoctor) [reply html/ajax_table.php:28]: just a thought, why not directly switch to filter_input on old code segments?
https://www.php.net/manual/en/function.filter-input.php

Comment (Jellyfrog) [reply html/ajax_table.php:28]: Thats not a protection against sql injections

Comment (SourceDoctor) [reply html/ajax_table.php:28]: it validates Types and escapes Strings ... thats what's the sanitize filte does
https://www.php.net/manual/en/filter.filters.sanitize.php

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-66-release-changelog-july-2020/12953/1

**Pull request merged**